next.config.js

```diff
@@ -5,7 +5,7 @@ const localesDir = path.resolve("public/static/locales");
 module.exports = {
   i18n,
   sassOptions: {
-    includePaths: [path.join(__dirname, "styles")],
+    includePaths: [path.join(__dirname, "styles"), path.resolve("node_modules")],
```

Comment:
in order to use @import 'react-phone-input-2/lib/style.css'; 
if you have a better approach then i'm all for it.

Reply:
We cannot include this in the path as it would significantly increase bundle size. I would say if this is a must please include the specific package rather than the entire node_modules